templates/download.html.twig

```diff
@@ -41,6 +41,16 @@ php -r "unlink('composer-setup.php');"</pre>
         <li>Remove the installer</li>
     </ul>
 
+    <p>Most likely, you want to put the <code>composer.phar</code> into a directory on your PATH, so you can simply call <code>composer</code> from any directory (<em>Global install</em>):
+    </p>
+    <pre class="language-bash">sudo mv composer.phar /usr/local/bin/composer</pre>
+    <p>
+        For details, see
+        <a href="/doc/00-intro.md#globally"
+           title="See the instructions on how to install Composer globally"
+           aria-label="See the instruction on how to install Composer globally">Getting started: Global install</a>.
```

Comment:
I'd rather write that sentence as the actual text of the link instead of putting it only in an aria-label. To me, it is more clear for everyone

Reply:
Thanks for the fast feedback! Just updated my PR accordingly 😊